cmd/operator-sdk/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	// a Kubebuilder project as a way to opt into the new KB CLI.
 	// TODO: Make the new KB CLI the default, once the integration is complete
 	// and deprecate "operator-sdk new" from the old CLI.
-	if kbutil.IsConfigExist() {
+	if kbutil.HasProjectFile() {
```

Comment:
keep the nomenclature used in kube which also make more clear what actually it is cheking